eox_tagging/edxapp_accessors.py

```diff
@@ -52,16 +50,12 @@ def get_course_enrollment(**kwargs):
 
     if getattr(settings, "EOX_TAGGING_SKIP_VALIDATIONS", False):
         return object
-
-    enrollment, _ = get_enrollment(username=username, course_id=course_id)
     try:
-        # Temporary method to get CourseEnrollment objects. This should be a backend.
-        from student.models import CourseEnrollment
         user = get_user(**{"target_id": username})
         course_id = CourseKey.from_string(course_id)
         return CourseEnrollment.objects.get(user_id=user.id, course_id=course_id)
-    except ImportError:
-        return enrollment
+    except AttributeError:
+        return get_enrollment(username=username, course_id=course_id)
```

Comment:
In this case do you want to return a tuple ? Can we raise an exception instead, like BadConfiguration, missing requirement... etc. I don't like this, since we have a method which is not reliable, this can return a CourseEnrollment, a tuple or a dict and other methods should handle all those cases.

Reply:
I'll remove the try-except block